scripts/pin_the_slow_way.py

```diff
@@ -27,34 +27,42 @@
 
 # The version commented right after the comma is the one available for **Linux** in **defaults**.
 # That version is assumed to be the same in all platforms and the one used to build all packages.
-# The only known incompatibilities at the moment are jpeg 8 vs 9, and icu 54.1 vs 58.*.
+# BE VERY CAREFUL when updating pinnings. Make sure to check https://abi-laboratory.pro/tracker/
+# for any incompatibility between versions.
+# Also be aware that currently the following pinnings are used for both build and run meta.yaml
+# sections. Defining the same pinning for both sections means we need to be more restrictive.
+#
+# For example, we used to define gmp as 'gmp >=5.0.1,<7'. This is wrong as it allows a package
+# to be built against gmp 6.1.0 but run with 5.0.1.
+# As showed by https://abi-laboratory.pro/tracker/timeline/gmp/ binaries built with 6.1.0 are not
+# compatible with 5.0.1 (the other way around would be true).
 pinned = {
           'boost': 'boost 1.64.*',  # 1.61.0
           'boost-cpp': 'boost-cpp 1.64.*',  # NA
           'bzip2': 'bzip2 1.0.*',  # 1.0.6
           'cairo': 'cairo 1.14.*',  # 1.12.18
-          'ffmpeg': 'ffmpeg 2.8.*',  # NA
+          'ffmpeg': 'ffmpeg >=3.2.3,<3.2.6',  # NA
           'fontconfig': 'fontconfig 2.12.*',  # 2.12.1
-          'freetype': 'freetype 2.7|2.7.*',  # 2.5.5
+          'freetype': 'freetype 2.7',  # 2.5.5
```

Comment:
The `|2.7.*` was there in case a patch version came out. Though it shouldn't matter much.